lib/perl/Genome/Model/Tools/Transcriptome/ErccMapUnaligned.pm

```diff
@@ -0,0 +1,367 @@
+package Genome::Model::Tools::Transcriptome::ErccMapUnaligned;
+
+use strict;
+use warnings;
+
+use Genome;
+use Path::Class;
+
+class Genome::Model::Tools::Transcriptome::ErccMapUnaligned {
+    is => 'Command::V2',
+    has_input => [
+        bam_file => {
+            is => 'Text',
+            doc => 'An aligned BAM file to a reference genome without the ERCC transcripts spiked in.',
+        },
+        ercc_fasta_file => {
+            is => 'Text',
+            doc => 'The FASTA format sequence for all ERCC transcripts.',
+            example_values => ['/gscmnt/gc13001/info/model_data/jwalker_scratch/ERCC/ERCC92.fa'],
+        },
+        ercc_spike_in_file => {
+            is => 'Text',
+            doc => 'The control analysis file provided by Agilent for the ERCC spike-in.',
+            example_values => ['/gscmnt/gc13001/info/model_data/jwalker_scratch/ERCC/ERCC_Controls_Analysis.txt'],
+        },
+        ercc_spike_in_mix => {
+            is => 'Integer',
+            doc => 'The expected ERCC spike-in mix.',
+            valid_values => [ '1', '2'],
+        },
+        samtools_version => {
+            is => 'Text',
```

Comment:
This could be `Float` instead of `Text`.  Since it has a default it probably doesn't also need an example.